src/index.ts

```diff
@@ -1 +1,51 @@
-//Define class/functions here
\ No newline at end of file
+export interface IEvent {
+  readonly timestamp: number,
+  readonly eventType: eventTypeState,
+  value?: number
+}
+export type eventTypeState = 'newMessage' | 'screenShot' | 'view'
```

Comment:
```suggestion
export type EventType = 'newMessage' | 'screenShot' | 'view'
```

types/interfaces should be capitalized

Reply:
Done